btcpy/structs/transaction.py

```diff
@@ -188,15 +188,19 @@ class TxOut(Immutable, HexSerializable, Jsonizable):
     """
 
     @classmethod
-    def from_json(cls, dic):
-        return cls(int(Decimal(dic['value']) * Constants.get('to_unit')),
-                   dic['n'],
-                   ScriptBuilder.identify(bytearray(unhexlify(dic['scriptPubKey']['hex']))))
-
-    def __init__(self, value: int, n: int, script_pubkey: ScriptPubKey):
+    def from_json(cls, dic, network):
+        return cls(
+            int(Decimal(dic['value']) * network.decimals),
```

Comment:
Please check this commit: https://github.com/PeerAssets/btcpy/commit/73f22bf998fc5bbafa29d28045a36acbb75cc710

You need to move from "network.decimals" to appropriate new entry in Constants.

Reply:
:+1: will patch up